### PR TITLE
CNV-32568: Fixing Hardware Devices table in Compute -> Hardware devices

### PR DESCRIPTION
**Pull request by DanaOrr**
## 📝 Description

The Hardware Devices table in Compute -> Hardware Devices shows nothing.
Before the current change, the table was the same as the table that was in the vm overview. 
I changed the table and added the correct values ​​that should be there.

## 🎥 Demo
Before:
gpuBefore

After:
gpuAfter

### CODE REVIEW

File: src/utils/components/HardwareDevices/HardwareDevicesPageTable.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  loaded,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const columns = useHardwareDevicesPageColumns();
+
+  return (
+    <VirtualizedTable
+      columns={columns}
+      data={devices}
+      EmptyMsg={() => <div>{t('Not available')} </div>}
```
Comment (avivtur) [inline src/utils/components/HardwareDevices/HardwareDevicesPageTable.tsx:28]: div element is unnecessary, change it to Fragment (<></>) and is the space between text and closing tag intentional?

**Review by avivtur — COMMENTED** [review]
Looks good, I left one little comment

**Comment by avivtur** [comment]
> Before: gpuBefore

about the column name in the VM details page, Resource name should be just Name (resourceName is an actual property represented by "Hardware device name" column and using it as the column name for the name, is confusing to me) please change that as well

**Comment by metalice** [comment]
> > Before: gpuBefore
> >       
> >         [
> >           
> >         
> >             gpuBefore
> >           ](https://user-images.githubusercontent.com/61961469/266828720-64e87899-cd0e-43d8-8252-c4f5508a242f.gif)
> >         
> >         
> >           
> >             
> >               
> >             
> >             
> >               
> >               
> >             
> >           
> >           [
> >             
> >               
> >             
> >           ](https://user-images.githubusercontent.com/61961469/266828720-64e87899-cd0e-43d8-8252-c4f5508a242f.gif)
> 
> about the column name in the VM details page, "Resource name" should be just "Name" (`resourceName` is an actual property represented by "Hardware device name" column, and using it as the column label for the device's name, is confusing to me) please change that as well

hmm im not sure.. in the device modal we are adding a name of our own, so calling it only a name here might be confusing :)  WDYT?

File: src/utils/components/HardwareDevices/HardwareDevicesPageTable.tsx (hunk 1 above)
Comment (DanaOrr) [reply src/utils/components/HardwareDevices/HardwareDevicesPageTable.tsx]: The space at the end isn't intentional.
But maybe it's better if I use MutedTextSpan?

**Review by avivtur — APPROVED** [review]

**Pull request merged**